_overviews/scala3-macros/tutorial/reflection.md

```diff
@@ -75,6 +98,47 @@ def g[T: Type](using Quotes) =
   ...
 ```
 
+## Symbols
+
+The APIs of `Term` and `TypeTree` are relatively *closed* in the sense that methods produce and accept values
+whose types are defined in the API. You might notice however the presence of `Symbol`s which identify definitions.
+
+Each `Term` or `TypeRepr` (therefore `Expr` and `Type`) have an associated symbol.
+`Symbol` exposes and is used by many useful methods for example:
+
+ - `declaredFields` and `declaredMethods` allow you to iterate on the fields and members defined inside a symbol
+ - `flags` allow you to check multiple properties of a symbol
+ - `companionObject` and `companionModule` provide a way to jump to and from the companion object/class.
+ - `TypeRepr.baseClasses` returns the list of symbols of classes extended by a type. 
+ - `Symbol.pos` gives you access to the position where the symbol is defined, the source code of the definition
+ and even the filename where the symbol is defined.
+ - and many useful others that you can find in `SymbolMethods`
+
+### To Symbol and back
+
+ - `TypeRepr.typeSymbol` returns the symbol of the type
+ - `TypeRepr.termSymbol` returns the symbol of the term associated to the type.
+ - `TypeRepr.memberType(symbol)` returns the `TypeRepr` of the provided symbol
+ - `TypeRepr.of[T].typeSymbol` is the recommended way to obtain a `Symbol` given a `Type[T]`
```

Comment:
Does it overlap with `TypeRepr.typeSymbol` mentioned in the first item?

Reply:
Yes it is a bit redundant, but it shows how to go quickly from `Type` to `Symbol`

Reply:
In that case, should we perhaps inline this bullet point into the first one?

> `TypeRepr.typeSymbol` returns the symbol of the type represented by `TypeRepr`. The recommended way to obtain a `Symbol` given a `Type[T]` is `TypeRepr.of[T].typeSymbol`.